src/coreclr/nativeaot/BuildIntegration/Microsoft.DotNet.ILCompiler.SingleEntry.targets

```diff
@@ -53,23 +53,42 @@
     </KnownILCompilerPack>
   </ItemGroup>
 
+  <PropertyGroup Condition="'$(TargetArchitecture)' == 'wasm'">
+    <NeedNativePublishSupportForLLVM>true</NeedNativePublishSupportForLLVM>
+  </PropertyGroup>
```

Comment:
`NeedNativePublishSupportForSDK6` will be deleted in https://github.com/dotnet/runtime/pull/82568 at which point this will conflict a lot...

I think we should just unconditionally override `ImportRuntimeIlcPackageTarget`:
```xml
  <!-- NativeAOT-LLVM: override ImportRuntimeIlcPackageTarget to not depend on the SDK-provided ResolvedILCompilerPack item -->
  <PropertyGroup>
    <!-- EmitLegacyAssetsFileItems tells the SDK to generate PackageDefinitions -->
    <EmitLegacyAssetsFileItems>true</EmitLegacyAssetsFileItems>
  </PropertyGroup>

  <Target Name="ImportRuntimeIlcPackageTarget" Condition="'$(BuildingFrameworkLibrary)' != 'true'" DependsOnTargets="$(ImportRuntimeIlcPackageTargetDependsOn)" BeforeTargets="Publish">
    <Error Condition="'@(PackageDefinitions)' == ''" Text="The PackageDefinitions ItemGroup is required for target ImportRuntimeIlcPackageTarget" />

    <PropertyGroup>
      <RuntimePackagePath Condition="'%(PackageDefinitions.Name)' == '$(RuntimeIlcPackageName)'">%(PackageDefinitions.ResolvedPath)</RuntimePackagePath>
      <IlcHostPackagePath Condition="'%(PackageDefinitions.Name)' == '$(IlcHostPackageName)'">%(PackageDefinitions.ResolvedPath)</IlcHostPackagePath>
    </PropertyGroup>

  </Target>
```
Before importing `Microsoft.NETCore.Native.targets`.

Reply:
Thanks, have tried that and looks simpler.